README.Rmd

```diff
@@ -33,6 +33,23 @@ You can install development version from [GitHub](https://github.com/) with:
 devtools::install_github("2DegreesInvesting/r2dii.climate.stress.test")
 ```
 
+Tu run most of the analyses, you will need additional auxiliary data, such as
+scenario data. The specific requirements are outlined below. **TODO**
+
+Before running the code on a local machine, set the path to the data files
+either as an Environment Variable or an R option using:
```

Comment:
might be good to specify explicitly if this var/value is required to be an **absolute** path, **relative** path, or either.... and likewise whether the return value can be expected to be one or both

Reply:
fair point